src/components/editor/StartStopBtnToolbar.tsx

```diff
@@ -0,0 +1,26 @@
+import { useState } from "react";
+import PlayArrowIcon from '@mui/icons-material/PlayArrow';
+import PauseIcon from '@mui/icons-material/Pause';
+import { grey } from '@mui/material/colors';
+
+function StartStopBtnToolbar() {
+    const [running, setRunning] = useState(false);
+
+    const handleToggle = (): void => {
+        setRunning((prev) => !prev);
```

Comment:
`setRunning` lokalnie jest ok, ale caly edytor musi wiedziec ze jest w trybie gry. spojrz na `EditorContext`. chcemy to trzymac globalnie. sprobuj dodac tam pole typu `running` i to modyfikuj. ten `EditorContext` bedzie w przyszlosci do wyjebania bo jest chujowy, ale na razie niech to tam bedzie